src/main/java/io/confluent/connect/jdbc/source/BulkTableQuerier.java

```diff
@@ -41,25 +42,33 @@ public BulkTableQuerier(
       DatabaseDialect dialect,
       QueryMode mode,
       String name,
-      String topicPrefix
+      String topicPrefix,
+      String suffix
   ) {
-    super(dialect, mode, name, topicPrefix);
+    super(dialect, mode, name, topicPrefix, suffix);
   }
 
   @Override
   protected void createPreparedStatement(Connection db) throws SQLException {
+    ExpressionBuilder builder = dialect.expressionBuilder();  
     switch (mode) {
       case TABLE:
-        String queryStr = dialect.expressionBuilder().append("SELECT * FROM ")
-                                 .append(tableId).toString();
+        builder.append("SELECT * FROM ").append(tableId);
+        addSuffixIfPresent(builder);
+
+        String queryStr = builder.toString();
+        
         recordQuery(queryStr);
         log.debug("{} prepared SQL query: {}", this, queryStr);
         stmt = dialect.createPreparedStatement(db, queryStr);
         break;
       case QUERY:
-        recordQuery(query);
-        log.debug("{} prepared SQL query: {}", this, query);
-        stmt = dialect.createPreparedStatement(db, query);
+        addSuffixIfPresent(builder);
+        queryStr = query + builder.toString();
```

Comment:
Can you call `builder.append(query);` rather than using string concatenation?

Reply:
Sure. I did it because previous version (in that case) didn't push query through builder.